libkbfs/folder_block_ops.go

```diff
@@ -1863,6 +1863,15 @@ func (fbo *folderBlockOps) startSyncWrite(ctx context.Context,
 		dirtyDe = &de
 	}
 
+	// Leave a copy of the syncOp in `unrefCache`, since it may be
```

Comment:
The real issue is that the op in `md` and the op in `unrefCache` can't be the same.  My comment is actually somewhat wrong -- the one in `syncState` and the one in `unrefCache` are always going to be the same, because `si` is a pointer.

However, during the course of `startSyncWrite()`, blocks are added to the op via the `md` that need to be reflected in `syncState.si.op`.  It's only at the end of the function that we can deep copy it away from the `unrefCache` one.  At that point, replacing the one in the `md` would require some surgery to the root metadata Ops list.  It'll be a little uglier that this, but if you think that's clearer I'll do it after standup.